constantine/eth_verkle_ipa/common_utils.nim

```diff
@@ -122,61 +128,3 @@ func computeNumRounds*(res: var uint32, vectorSize: SomeUnsignedInt)=
   debug: doAssert isP2 == true, "not a power of 2, hence not a valid inputs"
 
   res = uint32(log2_vartime(vectorSize))
-
-# ############################################################
-#
-#                   Pedersen Commitment
-#
-# ############################################################
-
-func pedersen_commit_varbasis*[EC_P] (res: var EC_P, groupPoints: openArray[EC_P], g: int,  polynomial: openArray[Fr], n: int)=
-  # This Pedersen Commitment function shall be used in specifically the Split scalars
-  # and Split points that are used in the IPA polynomial
-
-  # Further reference refer to this https://dankradfeist.de/ethereum/2021/07/27/inner-product-arguments.html
-  debug: doAssert groupPoints.len == polynomial.len, "Group Elements and Polynomials should be having the same length!"
-  var poly_big = newSeq[matchingOrderBigInt(Banderwagon)](n)
-  for i in 0 ..< n:
-    poly_big[i] = polynomial[i].toBig()
-
-  var groupPoints_aff = newSeq[EC_P_Aff](g)
-  for i in 0 ..< g:
-    groupPoints_aff[i].affine(groupPoints[i])
-
-  res.multiScalarMul_reference_vartime(poly_big, groupPoints_aff)
```

Comment:
MSM now supports field as input and does not need to convert to bigint explicitly.

Also, converting to affine coordinates is not necessary as we're always committing to a subset of the CRS and the CRS can be stored in affine coordinates.